### DISCUSSION TITLE
Hypixel API & Pulling items/Skyblock Inventories

### DISCUSSION THREAD

**Discussion by JEZEBEL20** [HypixelDev/PublicAPI#623] [Q&A]
This is a brief question but I wonder if its possible. Ive started my journey learning Java along side Kotlin; I would subsequently like my first project to be a private one for the time being, and that being a mod or bot that can pull the inventory of any player that has a specific item. 

Previously this was done with a mod such as iTEM but I would like to make it more private and perhaps even more efficient.

- **Overview** : basically if you ask the bot or in game mod, (preferably Id like to do both), for a specific item, let say you wanted to know any player that has a clover helmet you’d enter the search query for an item id then it would reply back with every player that has the item within their inventory/storage/enderchest/vault/ect

- **Purpose** : the general purpose of the mod would be for documenting items whereabouts. It’d be nice not to lose track of illusive and items that are on the brink of extinction, and seeing who has them can revitalize certain aspects.

In summary, all I’m essentially asking is whether creating a mod that can pull item data, location, and player information would be possible using the Hypixel API that was updated in the summer of 2023. Thank you for reading and hopefully responding, take care.
